salt/sciencebeam-texture/config/home-deployuser-sciencebeam-texture-smoke_tests.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 . /opt/smoke.sh/smoke.sh
 
+echo "Healthchecks"
+docker-wait-healthy sciencebeamtexture_texture_1
```

Comment:
Where is the name `sciencebeamtexture_texture_1` coming from?

Reply:
`sciencebeamtexture` is derived from `COMPOSE_PROJECT_NAME`, `texture` is the `service` in `docker-compose.yml` and `1` is the progressive number in case you use `docker-compose run` to create more.